modules/tensor_mechanics/src/materials/ComputeReturnMappingStress.C

```diff
@@ -77,10 +92,18 @@ ComputeReturnMappingStress::computeQpStress()
   RankTwoTensor strain_increment = _strain_increment[_qp];
   RankTwoTensor stress_new;
   updateQpStress(strain_increment, stress_new);
-  _elastic_strain[_qp] = _rotation_increment[_qp] * (strain_increment + _elastic_strain_old[_qp]) *
-                         _rotation_increment[_qp].transpose();
-  _stress[_qp] = _rotation_increment[_qp] * stress_new * _rotation_increment[_qp].transpose();
 
+  if (_perform_finite_strain_rotations)
+  {
+    _elastic_strain[_qp] = _rotation_increment[_qp] *
```

Comment:
We inherit from ComputeFiniteStrainElasticStress, which does the rotations on stress, but not elastic strain.  But we override ComputeFiniteStrainElasticStress::computeQpStress anyway, so we have to do the rotation here.

Yes, i wanted to make the rotation optional because of small strains.  i'll keep my eyes out for any trickiness to do with small vs finite strains.